scripts/e2e-email/validate-email.js

```diff
@@ -40,6 +40,12 @@ var messageContentChecks = [
     xheaders: [],
   },
   {
+    subject: 'New sign-in to Sync',
```

Comment:
So, the sequence is create and account, verify it, sign in as a new login, then change password, forget password, lock and unlock acount. So, the 3rd email received will be 'New sign-in to Sync'.